pkg/handler/handler.go

```diff
@@ -41,6 +42,15 @@ type podUpdateSettings struct {
 	useRegionalSTS bool
 }
 
+var mutations = prometheus.NewCounter(prometheus.CounterOpts{
+	Name: "mutations",
```

Comment:
by convention the name should have "_total" suffix:
...should have a suffix describing the unit, in plural form. Note that an accumulating count has total as a suffix, in addition to the unit if applicable. 
https://prometheus.io/docs/practices/naming/